### PR TITLE
Merge Volley master into mopub-volley

### PR DESCRIPTION
**Pull request by bichenwang**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -5,21 +5,154 @@
...
+// Custom properties for shadowing the library
+ext.tmp = "build/shadow"
+ext.buildTmp = tmp + "/src"
+ext.oldpkg = "com.android.volley"
+ext.newpkg = "com.mopub.volley"
 
 android {
-    compileSdkVersion 19
-    buildToolsVersion = '21.1.0'
+    compileSdkVersion 25
```
Comment (uchitel) [inline build.gradle]: Are we still on v. 25?

File: build.gradle [hunk 2]
```diff
@@ -5,21 +5,154 @@
...
+    useLibrary 'org.apache.http.legacy'
+
+    defaultConfig {
+        consumerProguardFiles 'consumer-proguard-rules.pro'
+    }
+
+    sourceSets {
+        defaultConfig {
+            minSdkVersion 21
+            targetSdkVersion 25
```
Comment (uchitel) [inline build.gradle]: Version number.

File: build.gradle [hunk 3]
```diff
@@ -5,21 +5,154 @@
...
+            versionCode 1
+            versionName "1.0"
```
Comment (uchitel) [inline build.gradle]: Do we want to increase version number?

File: build.gradle [hunk 4]
```diff
@@ -5,21 +5,154 @@
...
+    id "com.jfrog.bintray" version "1.0"
+}
+
+repositories {
+    jcenter()
+}
+
+task wrapper(type: Wrapper) {
+    gradleVersion = '2.1'
+}
```
Comment (uchitel) [inline build.gradle]: I believe it has to be 3.3.

File: build.gradle (hunk 3 above)
Comment (bichenwang) [reply build.gradle]: oh yeah, sure

**Comment by bichenwang** [comment]
Nice. "These conflicts are too complex to resolve in the web editor." So I think I'm just going to create a branch mopub-2.0 and go off there. no reason to merge into the mopub branch. One sec while I push my latest changes. Also, google volley is on 1.1.1, but the maven repo on bintray is still on 1.1.0. So I'm just going to call this mopub-volley 2.0.0

**Comment by bichenwang** [comment]
Pushed changes to the mopub-volley-2.0.0 branch. closing this pull request.

**Pull request closed without merging**